go/vt/vtgate/vcursor_impl.go

```diff
@@ -114,10 +112,10 @@ func (vc *vcursorImpl) ExecuteVSchema(keyspace string, vschemaDDL *sqlparser.Alt
 		return vterrors.Errorf(vtrpcpb.Code_INTERNAL, "vschema not loaded")
 	}
 
-	allowed := vschemaacl.Authorized(callerid.ImmediateCallerIDFromContext(vc.ctx))
+	user := callerid.ImmediateCallerIDFromContext(vc.ctx)
+	allowed := vschemaacl.Authorized(user)
 	if !allowed {
-		return vterrors.Errorf(vtrpcpb.Code_PERMISSION_DENIED, "not authorized to perform vschema operations")
-
+		return vterrors.NewErrorf(vtrpcpb.Code_PERMISSION_DENIED, vterrors.AccessDeniedError, "User '%s' to perform vschema operations", user.GetUsername())
```

Comment:
this message looks a bit weird to me. should it say: `User '%s' is not authorized to perform vschema operations`